src/components/Button/ButtonLink/styled.ts

```diff
@@ -0,0 +1,7 @@
+import styled from 'styled-components'
+
+export const ButtonCover = styled.div`
+	& a {
+		text-decoration: none;
+	}
+`
```

Comment:
You could style the `Link` component directly instead of wrapping it in another component.
```suggestion
import { Link } from 'react-router-dom'

export const StyledLink = styled(Link)`
	text-decoration: none;
`
```